src/OidcClientSettings.ts

```diff
@@ -168,6 +173,7 @@ export class OidcClientSettingsStore {
 
     public readonly revokeTokenAdditionalContentTypes?: string[];
     public readonly fetchRequestCredentials: RequestCredentials;
+    public readonly refreshTokenAllowableScopes: string | undefined;
```

Comment:
I am think about a cosmetic change, maybe `refreshTokenAllowedScope` is a better name?

- without s at the end to match `scope?: string;`
- Allowable vs Allowed

Reply:
Yes, I see why `refreshTokenAllowedScope` makes the naming more consistent. Just pushed the new name.